profile/isa_ro_crate_mapping.md

```diff
@@ -0,0 +1,157 @@
+| ISA term | schema term | Explanation |
```

Comment:
Initial header for the markdown and one sentence, smt like: This is the mapping between isa json types defined in https://isa-specs.readthedocs.io/en/latest/isajson.html and isa-ro-crate-json profile defined in https://github.com/nfdi4plants/isa-ro-crate-profile/blob/main/profile/isa_ro_crate.md .